docs/amdgpu_kernel_optimization_guide.md

```diff
@@ -136,12 +136,13 @@ split into 3 general groups:
 * AGPRs: Matrix accumulation vector registers (each thread holds a different
   value). Up to 256 AGPRs per thread on MI300.
 
-In total, the VGPRs within a CU hold 64 kB data (64 threads * 256 VGPRs * 4 B).
+VGPRs and AGPRs share the same register file: 512 registers * 64 threads per
+SIMD.
```

Comment:
If VGPRs is not sufficient, the amdgpu backend compiler can "spill" into AGPRs.

Reply:
Do you have a reference for this? I a haven't seen this before.